HopsanGenerator/src/generators/HopsanGeneratorBase.cpp

```diff
@@ -1127,6 +1127,11 @@ void hopsan::register_extra_components(hopsan::ComponentFactory* pComponentFacto
             }
 
             QString generatorErrorMessage;
+            QStringList newSourceFiles = lib.mExtraSourceFiles;
```

Comment:
why make a copy and then just take size of, could loop over range directly, or check size on original ?

Reply:
No idea :) I removed the unnecessary copy.